docs/deployment/customize-ci-jobs.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: customize-ci-jobs 
+title: Customize CI jobs 
+hide_title: true
+description: tk
+tags:
+  - Deployment
+---
+
+import DiffAwareScanning from "/src/components/reference/_diff-aware-scanning.mdx"
+import CiScheduling from "/src/components/reference/_ci-scheduling.mdx"
+import DeploymentJourney from "/src/components/concept/_deployment-journey.mdx"
+
+# Customize your CI job
+
+<DeploymentJourney />
+
+Customize a job's parameters to achieve the following goals: 
+
+* **Run Semgrep on a schedule**. Run full scans on mainline branches at the least intrusive time on developer teams.
+* **Run Semgrep with custom rules**. Apply rules specific to your organization's business goals and coding conventions.
+* **Run Semgrep when an event triggers**. Run Semgrep when a pull or merge request (PR or MR) is created. These event triggers or event hooks are dependent on your CI provider. 
+* **Run Semgrep on relevant files and blocks of code**. Configure Semgrep to ignore files and folders such as test files, configuration files, and files from other vendors.
+<!-- * **Configure a Semgrep CI job to pass even when any finding is detected**. By default, stand-alone configurations fail when any finding is detected. You can also configure Semgrep to pass CI jobs when findings are reported. -->
+* **Output, export, or save findings to a file**. Semgrep can save to a number of file formats, including SARIF and JSON.
```

Comment:
Shouldn't there be material on all of the items listed? I don't think I'm seeing it.